### DISCUSSION TITLE
Where is "start_http.bat" file?

### DISCUSSION THREAD

**Discussion by underdogme95** [w-okada/voice-changer#725] [Q&A]
I just downloaded the archive, but I can't find the file to run at all. Windows user.

**Comment by FernThatCodes** [comment]
First you need to unzip the file (you can find tutorials on YouTube on how to do that), inside the zip file is a folder, open the folder and scroll down (it might be somewhere down in the list) If none of that works try following this tutorial (https://youtu.be/_JXbvSTGPoo)
